rag-applications/remio-personal-knowledge-agent/app.py

```
@@ -0,0 +1,107 @@
+REMIO_URL = "https://remio.ai/"
+
+
+def run_remio(args):
+    if not shutil.which("remio"):
+        return {
+            "ok": False,
+            "error": "Remio CLI was not found. Install or open Remio from https://remio.ai/.",
+        }
+
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)